### PR TITLE
parsing error messages returned to cli

### PR DESCRIPTION
**Pull request by Akyiaa**
Error messages reported back to CLI are not being parsed correctly
Story: https://github.com/galasa-dev/projectmanagement/issues/1692

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: pkg/errors/galasaAPIErrorsArray.go [hunk 1]
```diff
@@ -24,13 +24,23 @@ func NewGalasaApiErrorsArray(body []byte) (*GalasaAPIErrorsArray, error) {
 
 	errorsGathered := new(GalasaAPIErrorsArray)
 
-	err = json.Unmarshal(body, &jsonArray)
+	//convert payload into string, check the first char to see if it is an array
+	stringBody := string(body)
+	if string(stringBody[0]) == "[" {
```
Comment (techcobweb) [inline pkg/errors/galasaAPIErrorsArray.go:29]: There must be a better way of doing this.
ie: Parsing the json, then asking the json whether it's a string array or an object ?

**Review by techcobweb — CHANGES_REQUESTED** [review]
See my comments,

**Comment by galasa-team** [comment]
Build successful

**Review by techcobweb — APPROVED** [review]

**Pull request merged**